From f9f6831f8c40b4e3426820ac8b23b27494fd9f84 Mon Sep 17 00:00:00 2001
From: Vitaly Zaitsev <vitaly@easycoding.org>
Date: Mon, 9 Apr 2018 16:57:13 +0200
Subject: [PATCH 2/3] Changed default Telegram Desktop's test API keys to
 obtained specially for Fedora/RPMFusion. We cannot hide them because we need
 to provide reproducible builds.

---
 Telegram/SourceFiles/config.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Telegram/SourceFiles/config.h b/Telegram/SourceFiles/config.h
index 863d0280e..255afd80c 100644
--- a/Telegram/SourceFiles/config.h
+++ b/Telegram/SourceFiles/config.h
@@ -197,8 +197,8 @@ w/CVnbwQOw0g5GBwwFV3r0uTTvy44xx8XXxk+Qknu4eBCsmrAFNnAgMBAAE=\n\
 #ifdef CUSTOM_API_ID
 #include "../../../TelegramPrivate/custom_api_id.h" // Custom API id and API hash
 #else
-static const int32 ApiId = 17349;
-static const char *ApiHash = "344583e45741c457fe1862106095a5eb";
+static const int32 ApiId = 208164;
+static const char *ApiHash = "dfbe1bc42dc9d20507e17d1814cc2f0a";
 #endif
 
 #if Q_BYTE_ORDER == Q_BIG_ENDIAN
-- 
2.17.1

